pkg/controller/jobframework/reconciler.go

```diff
@@ -643,26 +645,23 @@ func extractPriorityFromPodSets(podSets []kueue.PodSet) string {
 	return ""
 }
 
-type PodSetInfo struct {
-	Name         string            `json:"name"`
-	NodeSelector map[string]string `json:"nodeSelector"`
-	Count        int32             `json:"count"`
-}
-
-// getPodSetsInfoFromAdmission will extract podSetsInfo and podSets count from admitted workloads.
-func (r *JobReconciler) getPodSetsInfoFromAdmission(ctx context.Context, w *kueue.Workload) ([]PodSetInfo, error) {
+// getPodSetsInfoFromStatus extracts podSetsInfos from workload status, based on
+// admission, and admission checks.
+func (r *JobReconciler) getPodSetsInfoFromStatus(ctx context.Context, w *kueue.Workload) ([]PodSetInfo, error) {
 	if len(w.Status.Admission.PodSetAssignments) == 0 {
 		return nil, nil
 	}
 
-	nodeSelectors := make([]PodSetInfo, len(w.Status.Admission.PodSetAssignments))
+	podsSetInfos := make([]PodSetInfo, len(w.Status.Admission.PodSetAssignments))
```

Comment:
Done

Reply:
Actually, I can see cases, `PodSetsInfo`:

- https://github.com/kubernetes-sigs/kueue/blob/c3cfc1827b05b65c0dbc8a56e1ee84e76810f9f1/pkg/controller/jobframework/reconciler.go#L790 
- https://github.com/kubernetes-sigs/kueue/blob/c3cfc1827b05b65c0dbc8a56e1ee84e76810f9f1/pkg/controller/jobframework/interface.go#L37

So, I think we should re-phrase these at another PR since this is out of the scope of this PR. 

Anyway, I'm ok with https://github.com/kubernetes-sigs/kueue/pull/1180/commits/e6d8c2b5338bad6d3563591396e622d40359375d .

Reply:
Ah, I see, I will open another PR to rename the two places you point out.

Reply:
I just pushed a commit to revert the unnecessary rename in interface.go